_episodes/l4_04_parameterized_tests.md

```diff
@@ -0,0 +1,353 @@
+---
+title: "Parameterised Tests using GoogleTest"
+teaching: TBC
+exercises: TBC
+questions:
+- "What is a parameterised test?"
+- "How to write your own parameterised tests using GoogleTest?"
+objectives:
+- "Understand why do we need a parameterised test."
+- "Understand how to create a parameterised test using GoogleTest."
+- "Understand the advantages of parameterised tests."
+- "Understand how to combine a test fixtures with parameterised test."
+keypoints:
```

Comment:
Again, the keypoints need more detail, rather than just re-stating the objectives

Reply:
Resolved in 62676045679e0e5473c573ba35db49643fcbe7ba